src/extensions/default/DebugCommands/main.js

```diff
@@ -105,6 +107,63 @@ define(function (require, exports, module) {
         window.open(window.location.href);
     }
     
+    function _mapCLToolsErrorCodeToString(errorCode) {
+        
+        var errorString;
+        switch (errorCode) {
+        case appshell.app.ERR_CL_TOOLS_RMFAILED:
+            errorString = Strings.ERROR_CLTOOLS_RMFAILED;
+            break;
+        case appshell.app.ERR_CL_TOOLS_MKDIRFAILED:
+            errorString = Strings.ERROR_CLTOOLS_MKDIRFAILED;
+            break;
+        case appshell.app.ERR_CL_TOOLS_SYMLINKFAILED:
+            errorString = Strings.ERROR_CLTOOLS_LNFAILED;
+            break;
+        case appshell.app.ERR_CL_TOOLS_SERVFAILED:
+            errorString = Strings.ERROR_CLTOOLS_SERVFAILED;
+            break;
+        case appshell.app.ERR_CL_TOOLS_NOTSUPPORTED:
+            errorString = Strings.ERROR_CLTOOLS_NOTSUPPORTED;
+            break;
+        default:
+            errorString = "Unknown Error.";
```

Comment:
I think we should do this. But there is no generic Unknown Error string which we can use. We might want to add a string. I would suggest we do this in the next release as we have already sent the strings for translation.